src/Workspaces/CoreTest/SolutionTests/SolutionTests.cs

```diff
@@ -2138,7 +2138,7 @@ public void RemoveAnalyzerReference()
             solution = solution.WithAnalyzerReferences([analyzerRef1, analyzerRef2]);
 
             var solution2 = solution.RemoveAnalyzerReference(analyzerRef1);
-            AssertEx.Equal([analyzerRef2], solution2.AnalyzerReferences);
+            AssertEx.Equal((AnalyzerReference[])[analyzerRef2], solution2.AnalyzerReferences);
```

Comment:
:'(

is assertex something owned by roslyn?  can we just fixup the signatures here so this works? (or make it do a runtime check to do the right thing)?